imod/mf6/npf.py

```diff
@@ -11,6 +11,18 @@
     IdentityNoDataSchema,
     IndexesSchema,
 )
+from imod.typing import GridDataArray
+
+
+def _dataarray_to_bool(griddataarray: GridDataArray) -> bool:
+    if griddataarray is None or griddataarray.values is None:
+        return False
+
+    if griddataarray.values.size != 1:
+        raise ValueError(
+            "Dataarray is not a single value. Unable to convert to boolean"
+        )
+    return griddataarray.values.item()
```

Comment:
you could add a check on the dtype of griddataarray. It seems like in the use cases, it should be an actual bool and not an integer or float (and if it is, there may be something wrong)

Reply:
Good point. Added the check